style.css

```diff
@@ -529,3 +588,605 @@ textarea::placeholder {
+    font-weight: 500;
+    font-size: 16px;
+    color: #7f8cff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 142px;
+  }
+
+  ul.about-social-icons {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 20px;
+    margin-left: 110px;
+    list-style: none;
+  }
+
+  .resumebtn {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    border: 1px solid #6070ff;
+    cursor: pointer;
+    border-radius: 8px;
+    padding: 12px;
+    background: #fff;
+    margin: 0 0 134px 142px;
+  }
+
+  .resumebtn:enabled {
+    background: #fff;
+    color: #7f8cff;
+  }
+
+  .resumebtn:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  .resumebtn:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .resumebtn:disabled {
+    background: #fff;
+    color: #5e6c84;
+  }
+
+  .skill-list {
+    display: flex;
+    align-items: center;
+    width: 684px;
+    margin: 142px 166px;
+  }
+
+  .lang {
+    width: 684px;
+  }
+
+  ul.lang-frame-skills {
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .lang-frame-skills > li {
+    list-style: none;
+  }
+
+  .lang-list {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+  }
+
+  .lang-list-skill {
+    display: flex;
+  }
+
+  .lang-list > img {
+    align-self: center;
+  }
+
+  .lang-list > p {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    margin-left: 12px;
+  }
+
+  .frame-skill > p {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    margin-left: 12px;
+    margin-right: 12px;
+  }
+
+  .frame-skill > img {
+    align-self: center;
+    margin-right: 85px;
+  }
+
+  .lang-list-skill > div > p {
+    margin: 16px 16px 16px 0;
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1vw;
+    line-height: 20px;
+    color: #253858;
+  }
+
+  .lang-list-skill > div {
+    display: flex;
+    flex-direction: column;
+    align-content: flex-start;
+    justify-content: center;
+    width: 8.5vw;
+    height: 8.3vw;
+  }
+
+  .lang-list-skill > div > img {
+    width: 3.3vw;
+  }
+
+  .frame-skill {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid #dfe1e6;
+    width: 684px;
+  }
+
+  .contact-me {
+    background-image: url(./images/contact\ form\ background\ shapes\ desktop.png);
+    background-color: #6070ff;
+    border-top-left-radius: 10%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-content: center;
+    width: 100%;
+  }
```

Comment:
- [ ] Kindly reposition the `background-image` of the _**form**_ section of your portfolio on the desktop view to look like that of the Figma template that you chose. 

### Your Design

![Screenshot5](https://user-images.githubusercontent.com/77106962/222580419-6cf0f14a-9c3a-4c06-b583-f70128fda29c.png)

### Figma Template

![Screenshot6](https://user-images.githubusercontent.com/77106962/222581291-641199e5-443a-4582-9a9f-5d9e2deec245.png)

Reply:
@Gambit142 I centered the contact me background image